src/parsers/optimized/ps/Header.cs

```diff
@@ -180,8 +180,12 @@ private void AfterDateAndDocTypeBeforeCite(IBlock block) {
             return;
         }
         if (NextLineIsCiteOnly()) {
-            WLine title = WDocTitle2.ConvertContents(line);
-            Enriched.Add(title);
+            if (TitleHasBeenFound()) {
```

Comment:
In practice, when might a title be found and when might it not?

I know the logic for `TitleHasBeenFound` is added but my unfamiliarity with the classes in the repo is causing me to struggle to understand what that relates to in an actual document. I did try following up the definitions up the chain but I think a quick call to clarify would help a lot when you get the chance @jurisdatum